static/app/views/userFeedback/index.tsx

```diff
@@ -70,12 +71,20 @@ class OrganizationUserFeedback extends DeprecatedAsyncView<Props, State> {
         : [];
   }
 
-  renderResults() {
-    const {organization} = this.props;
-
+  function StreamBody() {
+    if (isLoading) {
```

Comment:
We should add an `if (isError)` here and show a `<LoadingError />` in case the request fails